client/src/components/menubar/index.js

```diff
@@ -32,7 +32,9 @@ import UndoRedoReset from "./undoRedo";
   aboutLink: state.config?.links?.["about-dataset"],
   disableDiffexp: state.config?.parameters?.["disable-diffexp"] ?? false,
   diffexpMayBeSlow: state.config?.parameters?.["diffexp-may-be-slow"] ?? false,
-  showCentroidLabels: state.centroidLabels.showLabels
+  showCentroidLabels: state.centroidLabels.showLabels,
+  tosURL: state.config?.about_legal_tos,
```

Comment:
these are wrong - should be pulling from parameters.

Ie. `state.config?.parameters?.about_legal_tos`, etc.